### PR TITLE
Respect translations from "Other" tab in meetings editor

### LINKED ISSUE #245: add missing translations
on the meeting editor all strings for field labels in the other section can't be translated

Screen Shot 2019-11-30 at 7 17 04 PM

any field added to `_comdef_meetings_data`  with a `meetingid_bigint` of `0` will create a new field in editor as well as return it in a `GetSearchResults` meeting response. we should only be concerned with the fields populated on a stock server https://github.com/bmlt-enabled/bmlt-root-server/blob/master/main_server/local_server/install_wizard/sql_files/InitialMeetingsData.sql

```sql
INSERT INTO `na_comdef_meetings_data` (`meetingid_bigint`, `key`, `field_prompt`, `lang_enum`, `visibility`, `data_string`, `data_bigint`, `data_double`) VALUES ('0', 'boat_line', 'Boat Lines', 'en', NULL, 'Steamship Authority', NULL, NULL);`
INSERT INTO `na_comdef_meetings_data` (`meetingid_bigint`, `key`, `field_prompt`, `lang_enum`, `visibility`, `data_string`, `data_bigint`, `data_double`) VALUES ('512', 'boat_line', 'Boat Lines', 'en', NULL, 'Steamship Authority', NULL, NULL);
```

Screen Shot 2019-11-30 at 7 14 33 PM

### PR DESCRIPTION
**Pull request by pjaudiomv**
resolves #245

### CODE REVIEW

File: main_server/local_server/server_admin/c_comdef_admin_main_console.class.php [hunk 1]
```diff
@@ -1879,6 +1879,87 @@ public function return_single_meeting_other_template()
                 case 'location_postal_code_1':
                 case 'location_nation':
                     break;
+                case 'comments':
+                    $ret .= '<div class="bmlt_admin_one_line_in_a_form clear_both">';
```
Comment (jbraswell) [inline main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: I feel like all of these could be handled in one block using `$key`

Comment (pjaudiomv) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: I feel like your right

Comment (pjaudiomv) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: not sure how to do this while using localized strings and while still being able to allow custom fields, revert to default switch

Comment (pjaudiomv) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: unless I check if the localized key exists

Comment (jbraswell) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: Yes, if the key doesn’t exist then use what is in the db

Comment (pjaudiomv) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: ok I did those things and tested with localized strings and without

File: main_server/local_server/server_admin/c_comdef_admin_main_console.class.php [hunk 2]
```diff
@@ -22,9 +22,9 @@
 // #define ( '__NAWS_IMPORT__', 1 )    /* Uncomment to enable the NAWS import functionality. */
 
 /************************************************************************************************************//**
-    \class c_comdef_admin_main_console
-    \brief Controls display of the main BMLT administration console.
-****************************************************************************************************************/
+\class c_comdef_admin_main_console
```
Comment (jbraswell) [inline main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: Lot of whitespace diffs in this file bro

Comment (pjaudiomv) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: yea wtf super annoying

**Comment by pjaudiomv** [comment]
ok I like this

File: README.md [hunk 3]
```diff
@@ -57,6 +57,10 @@ For instructions on installing the root server, see [the page on installing a ne
 
 CHANGELIST
 ----------
+***Version 2.14.5* ** *- UNRELEASED*
+
+- Added Other tab from admin user interface to translatable strings.
```
Comment (jbraswell) [inline README.md]: Change to `The "Other" tab in the meeting editor user interface can now be translated.`

Then feel free to merge.

**Pull request merged**